Dev/servicemonitor.1m.py

```diff
@@ -58,71 +77,208 @@
 		},
 	]
 }
-# green: \x1b[42m
-# red: \x1b[41m
-# blue: \x1b[44m
-HEALTHY='\x1b[42m'
-UNHEALTHY='\x1b[41m'
-NC='\x1b[0m'
-
-def process_environments():
-	envs = SERVICES.keys()
-	result = {}
-	for env in envs:
-		result[env] = process_services(SERVICES[env])
-	return result
-
-def process_services(services):
-	servicesnames = [ x["name"] for x in services]
-	with Pool() as pool:
-		result = pool.map(call_service, services)
-	return dict(zip(servicesnames, result))
-
-def call_service(service):
-	method =  service["method"] if "method" in service.keys() else "GET"
-	data = service["body"] if "body" in service.keys() else None
-	headers = {"user-agent":"xbar"}
-	if "headers" in service.keys():
-		headers.update(service["headers"])
+
+# Max number of issues in the menu
+LIMIT_TOTAL_ISSUES=5
+# Recovery time in Hours
+RECOVERY_TIME=12
+
+ALLOW_NOTIFICATIONS = True
```

Comment:
These and `TITLE` above might be a good use case for [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html).